src/app/components/projectversion/projectversion-repo-list.html

```diff
@@ -37,6 +37,15 @@ <h1><mat-icon>class</mat-icon>Project <a class="title" [routerLink]="['/project'
         </td>
     </ng-container>
 
+    <ng-container matColumnDef="latest_release">
+        <th mat-header-cell *matHeaderCellDef>Latest Release</th>
+        <td mat-cell *matCellDef="let element" [routerLink]="['/project', projectversion.project_name, projectversion.name, 'repo', element.id]">
+            <div *ngIf="element.latest_release" style="display: flex; align-items: center; font-weight: bold;">
+                {{element.latest_release.version}}
```

Comment:
What if `element.laest_release` doesn't exist? It seems a possibility, given this code: https://github.com/molior-dbs/molior/pull/22/files#diff-a55367c6f8325b333ce41b5b5737f37ceb47a618502ce102443553200d35f0b0R268

If so, then won't `element.latest_release.version` raise an error?

Reply:
will add return None in molior API for such case

Reply:
I am not sure I understand exactly what you are proposing, but even if `element.latest_release` is None, still trying to access `element.latest_release.version` would raise an error.

I think it's fine for `element.latest_release` to either be missing or be None: but in either case the frontend code above should check `if element.latest_release then... else...`.

Reply:
ngIf="element.latest_release" when it is None no access to .version